cadquery/cq.py

```diff
@@ -1933,44 +1934,6 @@ def tangentArcPoint(
 
         return self.newObject([arc])
 
-    '''
-    def rotateAndCopy(self, matrix: Matrix) -> "Workplane":
```

Comment:
Why was this commented out and now removed?

Reply:
I just removed commented code. Looks like I might have commented it our by mistake in previous PRs - I'll revert this change.

Reply:
I did some brief archeology and now it is clear why I commented it out: it was calling the method `rotateShapes` which was not implemented (i.e. throwing a `NotImplementedError`). Hence I removed both.